tsconfig.json

```diff
@@ -0,0 +1,107 @@
+{
```

Comment:
this file can probably be trimmed. I used `tsc init` to create it.

Reply:
yes, can the comments be removed here?